specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-07-10/service.json

```diff
@@ -10468,6 +10476,13 @@
         "useManagedDisks": {
           "description": "A value indicating whether managed disks should be used during failover.",
           "type": "string"
+        },
+        "diskIdToDiskEncryptionMap": {
+          "description": "The dictionary of disk resource Id to disk encryption set ARM Id.",
```

Comment:
The use of "resource id" or "ARM id" is inconsistent throughout but not part of your change.  However, having a description that uses both doesn't make sense.  "resource ID" is my preferred terminology but at the least it should be consistent.

Reply:
sure, will revisit the descriptions.